code/modules/awaymissions/mission_code/wildwest.dm

```diff
@@ -80,19 +80,40 @@
 				user.mutations.Add(LASER)
 				user.mutations.Add(RESIST_COLD)
 				user.mutations.Add(XRAY)
-				user.dna.mutantrace = "shadow"
+				if(ishuman(user))
+					var/mob/living/carbon/human/human = user
+					if(human.species.name != "Shadow")
```

Comment:
glitchy adminbooze can fuck anything up. The code itself is sound, protecting against adminbooze runtimes is tilting against windmills.